quickwit/quickwit-indexing/src/split_store/local_split_store.rs

```diff
@@ -135,11 +135,7 @@ impl SplitFolder {
 }
 
 fn split_id_from_split_folder(dir_path: &Path) -> Option<&str> {
-    if !dir_path.is_dir() {
-        return None;
-    }
-    let split_id: &str = dir_path.file_name()?.to_str()?.strip_suffix(".split")?;
-    Some(split_id)
+    dir_path.file_name()?.to_str()?.strip_suffix(".split")
 }
```

Comment:
Im tempted to move this out of the function now, but it equally doesn't do any harm to be a bit more verbose I guess.